ibm_catalog.json

```diff
@@ -24,16 +24,21 @@
         "ai"
       ],
       "short_description": "Creates and configures an instance of IBM Watson Discovery.",
-      "long_description": "This architecture supports creating and configuring an instance of Watson Discovery.",
+      "long_description": "IBM Watson Discovery is an intelligent document processing engine designed to extract valuable insights from complex business documents. With Discovery, you can visually train AI models to deeply understand your content — including text, tables, and images — uncovering business value that may be hidden within your enterprise data. For more details on Watson Discovery, [refer here](https://www.ibm.com/products/watson-discovery).",
       "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-discovery/blob/main/README.md",
       "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-watsonx-discovery/main/images/watson-discovery.svg",
       "provider_name": "IBM",
       "features": [
         {
-          "title": "Creates an instance of IBM Watson Discovery",
-          "description": "Creates and configures an IBM Watson Discovery instance."
+          "title": "Watson Discovery",
+          "description": "Creates the Watson Discovery Instance. [Learn more](https://cloud.ibm.com/docs/discovery-data?topic=discovery-data-about)"
```

Comment:
Refer this - https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-data/blob/main/ibm_catalog.json#L30